build/yamls/antrea-aks.yml

```diff
@@ -2114,6 +2155,19 @@ spec:
                                   type: integer
                                   minimum: 0
                                   maximum: 255
+                            igmp:
+                              type: object
+                              properties:
+                                igmpType:
+                                  type: integer
+                                  # Only IGMP reports are igmpType in egress rules,
+                                  # 0x12 is IGMP report V1, 0x16 is IGMP report v2, 0x22 is IGMP report v3.
```

Comment:
1. Please unify case usage for v1, v2 and v3.
2. If I understand correctly, igmp Leave Group message is not covered in np?

Reply:
there is no code to indicate all report, unless we support empty IGMP type for all IGMP type

Reply:
done